tests/test_scalecomposite.py

```diff
@@ -118,6 +119,8 @@ def properties(self):
         return self.child.properties()
 
 
+# @dimod.testing.load_sampler_bqm_tests(ScaleComposite(ExactSolver()))
+# @dimod.testing.load_sampler_bqm_tests(ScaleComposite(dimod.NullSampler))
```

Comment:
Out of curiosity, why aren't these samplers tested here as well?

Reply:
They fail! There are two issues #629 #628 linked above.